setup.cfg

```diff
@@ -38,7 +38,7 @@ setup_requires =
     setuptools_scm
     pytest-runner
 install_requires =
-    click
+    click>7.0
```

Comment:
See https://github.com/pallets/click/blob/main/CHANGES.rst#version-71, specifically:

> Fix TypeError raised when using bool flags and specifying type=bool. [:issue:`1287`](https://github.com/pallets/click/blob/main/CHANGES.rst#id325)

The shield/local option will error due to this on `click<=7.0`.